contracts/validator-manager/MessageSpec.md

```diff
@@ -101,27 +159,27 @@ Specification:
                           +----------+
 ```
 
-### `SubnetValidatorWeightUpdateMessage`
+### `ValidationUptimeMessage`
+
+- Description: Provides a Validator's uptime for calculating staking rewards
 
-Description: Acknowledges a Validator weight update
+- Signed by: Subnet
```

Comment:
Not for this PR, but just a heads up that we'll be updating the terminology from subnet -> L1 soon, including for these message names and transaction types.

Subnets will refer to pre-Etna Subnets, where their validator set must also validate the primary network, making it a subset of the primary network validators.

L1s will refer to post-Etna sovereign networks, whose validators sets can be completely independent of the primary network.